pkg/graph/schema.resolvers.go

```diff
@@ -1180,8 +1179,42 @@ func (r *mutationResolver) DeleteTrbLeadOption(ctx context.Context, eua string)
 }
 
 // SendGRBReviewPresentationDeckReminderEmail is the resolver for the sendGRBReviewPresentationDeckReminderEmail field.
-func (r *mutationResolver) SendGRBReviewPresentationDeckReminderEmail(ctx context.Context, systemIntakeID uuid.UUID) (*string, error) {
-	panic(fmt.Errorf("not implemented: SendGRBReviewPresentationDeckReminderEmail - sendGRBReviewPresentationDeckReminderEmail"))
+func (r *mutationResolver) SendGRBReviewPresentationDeckReminderEmail(ctx context.Context, systemIntakeID uuid.UUID) (bool, error) {
+	intake, err := r.store.FetchSystemIntakeByID(ctx, systemIntakeID)
+	if err != nil {
+		return false, err
+	}
+
+	if intake.RequesterEmailAddress.Ptr() == nil {
+		return false, &apperrors.ResourceNotFoundError{Err: errors.New("no requester email address found")}
+	}
+
+	recipients := models.EmailNotificationRecipients{
+		RegularRecipientEmails: []models.EmailAddress{
+			models.NewEmailAddress(intake.RequesterEmailAddress.ValueOrZero()),
+		},
+		ShouldNotifyITGovernance: false,
+		ShouldNotifyITInvestment: false,
+	}
+
+	err = r.emailClient.SystemIntake.SendPresentationDeckUploadReminder(
+		ctx,
+		recipients,
+		intake.ID,
+		intake.ProjectName.ValueOrZero(),
+	)
+	if err != nil {
+		return false, err
+	}
+
+	curTime := time.Now()
+	intake.GrbPresentationDeckRequesterReminderEmailSentTime = &curTime
+	_, err = r.store.UpdateSystemIntakeNP(ctx, r.store, intake)
```

Comment:
the non-NP version of this is available if preferable

```suggestion
	_, err = r.store.UpdateSystemIntake(ctx, intake)
```

Reply:
Good suggestion. I didn't like this line.